buildspec.yml

```diff
@@ -25,22 +25,39 @@ phases:
       - tox -e py36
 
       # push docker images to ECR
-      - if-release-build ./scripts/publish.sh --version 1.11.1 --arch cpu
-      - if-release-build ./scripts/publish.sh --version 1.11.1 --arch gpu
-      - if-release-build ./scripts/publish.sh --version 1.12.0 --arch cpu
-      - if-release-build ./scripts/publish.sh --version 1.12.0 --arch gpu
-
-      # write images to manifest (used by deploy step)
-      - if-release-build create-manifest docker
-            --images
-              sagemaker-tensorflow-serving:1.11.1-cpu
-              sagemaker-tensorflow-serving:1.11.1-gpu
-              sagemaker-tensorflow-serving:1.12.0-cpu
-              sagemaker-tensorflow-serving:1.12.0-gpu
+      - |
+        if is-release-build; then
+          ./scripts/publish.sh --version 1.11.1 --arch cpu
+          ./scripts/publish.sh --version 1.11.1 --arch gpu
+          ./scripts/publish.sh --version 1.12.0 --arch cpu
+          ./scripts/publish.sh --version 1.12.0 --arch gpu
+        fi
+
+      # write deployment details to file
+      - |
+        if is-release-build; then
+          echo '[{
+          "repository": "sagemaker-tensorflow-serving",
+          "tags": [{
+            "source": "1.11.1-cpu",
```

Comment:
Just wondering, is there a reason we chose 1.11.1.

It looks like the training and the Python based TFS container both used 1.11.0

https://github.com/aws/sagemaker-tensorflow-container/tree/script-mode/docker/1.11.0
https://github.com/aws/sagemaker-tensorflow-container/tree/master/docker/1.11.0/final/py2

Reply:
1.11.1 is the actual version of tensorflow/tensorflow serving in this container. we had a previous release with 1.11.0, but aren't maintaining images for that version anymore.